doc/ref_model/chapters/chapter04.md

```diff
@@ -262,6 +263,32 @@ Table 4-10 shows performance measurement capabilities.
+ 
+For Accelerators and Offload functions with externally exposed differences in their capabilities or management functionality these differences must be clear through the management API either explicit for the differing functions or implicit through the use of a unique APIs.
+ 
+Regardless of exposed or internal different capabilities and characteristics, the operators generally require a choice of multiple implementations also for Accelerators and Offload function realization, which drive the need for ease of portability in between implementations and vendors.
+ 
+The following table of requirement are derived from the need of the VNF/CNF applications, Cloud Infrastructure and Telco Operators to have multiple realizations of HW Acceleration and Offload functions that could also be implemented through a software implementation when no special hardware is available. These should be taken into any Reference Architecture to ensure that the different implementations on the market are as aligned as possible in their interfaces and that HW Acceleration and Offload gets an efficient eco system of accelerators that compete on their technical merits and not through obscured or proprietary interfaces.
+
+**Table 4-12** shows Acceleration/Offload API Capabilities.
+
+<a name="Table4-12"></a>
+
+| Ref       | Acceleration/Offload API Capability | Unit   | Definition/Notes                                                                            |
+|-----------|--------------------------------------------|--------|---------------------------------------------------------------------------------------------|
+| e.api.001 | VNF/CNF usage of Accelerator standard i/f                           | Yes/No | VNF/CNF shall use abstracted standardized interfaces to the Acceleration/Offload function that enables multiple HW and SW implementation of the accelerated/offloaded function from multiple vendors in the Cloud Infrastructure                        |
+| e.api.002 | Virtualization Infrastructure SW usage of Accelerator standard i/f  | Yes/No | Virtualization Infrastructure SW shall use abstracted standardized interfaces to the HW-Acceleration/Offload  function that enables multiple HW and SW implementation in the HW Infrastructure Layer of the accelerated function from multiple vendors  |
+| e.api.003 | Accelerators offering standard i/f to HW Infra Layer                | Yes/No | Acceleration/Offload functions shall offer abstracted standardized interfaces for the Virtualization Infrastructure and VNF/CNF applications                                                                                                             |
```

Comment:
Can we combine the first 3 into a single requirement?

| e.api.001 | Accelerators offer standard i/f                 | Yes/No |  The Platform shall offer abstracted standardized interfaces for LCM of the Acceleration/Offload functions for use by the Cloud Infrastructure, VNF/CNF applications, and other users. |   

or something similar

Reply:
I have a hard time combining all of them, since they are different requirements on different entities.  Possibly 001 and 002 could be combined, but they will then need to be acted upon by different Anuket Groups (RI-x/RC-x and RA-x) and implementations (VNFs/CNFs as being the applications and Virtualizaton Infrastructure as being the Infrastructure internal layering requirements).
001: Mandate VNF/CNF to use abstracted API where they exist
002 Mandate SW Virtualization layer implementations (i.e. OpenStack and K8s) to use abstracted API where they exist
To your suggested text, it is far more than LCM interfaces we talk about here.
Why is it important to bundle or abstract the requirements? The risk is just that it become obfuscated.

Reply:
@TFredberg Can we simplify? A suggestion made in reqt. 001. BTW we use the word "must".

Reply:
You are suggesting other requirements through the change of text. So, yes, that points to that we must change the text, since it is not properly understood. 
Usage of the word Platform neither helps me, since that is an undefined entity here.

Can we first agree on what the requirement is, and then we can possibly word-smith it into a simple enough form that still is correct.
001: Intend to say that VNFs and CNFs shall be written and tested to support abstracted Acceleration/Offload "standard" interface when such once are available, even if there exist vendor or device-specific interfaces that might be more comprehensive or efficient. It shall be up to an application-external orchestrator which of the application supported accelerators or offload functions to actually deploy at any point of deployment.
002: Intend to say the same as 001 but shall be valid for the Virtualization SW inside the Cloud Infrastructure, which is just one of the internal layers inside the Cloud Infrastructure.